### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by idivyanshbansal**
Signed-off-by: Divyansh Bansal <divyansh.bansal25@gmail.com>

## Read then delete

- Make sure to use a concise title for the pull-request.
- Use #patch, #minor #majora or #none in the pull-request title to bump the corresponding version. Otherwise, the patch version
  will be bumped. [More details](https://github.com/marketplace/actions/github-tag-bump)

# TL;DR
_Please replace this text with a description of what this PR accomplishes._

## Type
- [x] Bug Fix
- [x] Feature
- [ ] Plugin

## Are all requirements met?

- [x] Code completed
- [x] Smoke tested
- [x] Unit tests added
- [x] Code documentation added
- [x] Any pending items have an associated Issue

## Complete description
_How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@
 ![Commit activity](https://img.shields.io/github/commit-activity/w/lyft/flytectl.svg?style=plastic)
 ![Commit since last release](https://img.shields.io/github/commits-since/lyft/flytectl/latest.svg?style=plastic)
 
-FlyteCTL is designed to be a portable and lightweight CLI to work with Flyte. It is written in Golang and can access [FlyteAdmin](https://github.com/flyteorg/flyteadmin/)—the control plane of Flyte.
+FlyteCTL was designed as a portable and lightweight command-line interface to work with Flyte. It is written in Golang and accesses [FlyteAdmin](https://github.com/flyteorg/flyteadmin/), the control plane for Flyte.
```
Comment (kumare3) [inline README.md:28]: Present perfect tense?

Comment (idivyanshbansal) [reply README.md:28]: The first sentence is in simple past tense and the second in the present tense in passive form.

**Review by kumare3 — APPROVED** [review]

**Review by kumare3 — APPROVED** [review]

**Comment by idivyanshbansal** [comment]
Thank you so much sir @kumare3 🤩🤩. Sir, please mark this as hacktoberfest-accepted.🥺🥺

**Comment by kumare3** [comment]
> Thank you so much sir @kumare3 🤩🤩. 

Absolutely- welcome to the community

**Pull request merged**